Client/src/App.jsx

```diff
@@ -27,6 +27,7 @@ import CreatePageSpeed from "./Pages/PageSpeed/CreatePageSpeed";
 import CreateNewMaintenanceWindow from "./Pages/Maintenance/CreateMaintenanceWindow";
 import PageSpeedDetails from "./Pages/PageSpeed/Details";
 import PageSpeedConfigure from "./Pages/PageSpeed/Configure";
+import ComponentText from "./Pages/ComponentText";
```

Comment:
This doesn't resolve and app won't run as a result, I assume this is from testing?  It can be removed

Reply:
Oh, it was for the purpose of test!
Forgot to remove it!